Objects/listobject.c

```diff
@@ -2824,7 +2824,8 @@ static PyObject *list_iter(PyObject *seq);
 static PyObject *list_subscript(PyListObject*, PyObject*);
 
 static PyMethodDef list_methods[] = {
-    {"__getitem__", (PyCFunction)list_subscript, METH_O|METH_COEXIST, "x.__getitem__(y) <==> x[y]"},
+    {"__getitem__", (PyCFunction)list_subscript, METH_O|METH_COEXIST,
+     "__getitem__($self, index, /)\n--\n\nReturn self[index]."},
```

Comment:
LGTM. But would you wrap the docstring with `PyDoc_STR("...")`?

Reply:
Sure! Strange, that it was not.